coriolis/osmorphing/manager.py

```diff
@@ -194,7 +194,7 @@ def morph_image(origin_provider, destination_provider, connection_info,
 
     import_os_morphing_tools = import_os_morphing_tools_cls(
         conn, os_root_dir, os_root_dev, hypervisor_type,
-        event_manager, detected_os_info)
+        event_manager, detected_os_info, osmorphing_info)
```

Comment:
Note that the Windows tools have not been updated so you'll get a guaranteed `TypeError` here for Windows morphers.